debs/xenial/install.sh

```diff
@@ -0,0 +1,136 @@
+    printf '\n '$(sha256sum Packages | cut --delimiter=' ' --fields=1)' %16d Packages' $(wc --bytes Packages | cut --delimiter=' ' --fields=1) >> Release
+
+    # Sign it using our dummy keys
+    chmod 700 ${ROOT}/deb-testing/.gnupg
+    chmod 600 ${ROOT}/deb-testing/.gnupg/*
+    sudo apt-key add ${ROOT}/deb-testing/.gnupg/pubring.gpg
+    gpg --batch --yes --clearsign --digest-algo SHA512 --homedir ${ROOT}/deb-testing/.gnupg -o InRelease Release
+}
+
+readonly remote_apt_keys=(
+    "https://packages.archivematica.org/1.7.x/key.asc"
+    "http://jenkins-ci.archivematica.org/repos/devel.key"
+    "https://packages.elasticsearch.org/GPG-KEY-elasticsearch"
+)
+
+apt_repositories=(
+    "deb [arch=amd64] http://packages.archivematica.org/1.7.x/ubuntu-externals xenial main"
+    "deb http://packages.elasticsearch.org/elasticsearch/1.7/debian stable main"
+)
+
+readonly packages=(
+    "elasticsearch"
+    "archivematica-storage-service"
+    "archivematica-mcp-server"
+    "archivematica-dashboard"
+    "archivematica-mcp-client"
+)
+
+readonly services=(
+    "elasticsearch"
+    "clamav-freshclam"
+    "clamav-daemon"
+    "fits"
+    "nginx"
+    "gearman-job-server"
+    "archivematica-mcp-server"
+    "archivematica-mcp-client"
+    "archivematica-storage-service"
+    "archivematica-dashboard"
+)
+
+sudo apt-get update
+sudo apt-get upgrade --yes
+sudo apt-get install --yes python
+
+if [ "${build_local_repository}" == "true" ] ; then
+    repo="${ROOT}/deb-testing/.repo"
+    create_local_repository "${repo}"
+    apt_repositories+=("deb file:${repo} ./")
+else
+    apt_repositories+=("deb http://jenkins-ci.archivematica.org/repos/apt/release-0.11-xenial/ ./")
+    apt_repositories+=("deb http://jenkins-ci.archivematica.org/repos/apt/release-1.7-xenial/ ./")
+fi
+
+for item in "${remote_apt_keys[@]}"; do
+    add_remote_apt_key "${item}"
+done
+
+for item in "${apt_repositories[@]}"; do
+    add-apt-repository "${item}"
+done
+
+sudo debconf-set-selections <<< 'mysql-server mysql-server/root_password password your_password'
+sudo debconf-set-selections <<< 'mysql-server mysql-server/root_password_again password your_password'
+sudo debconf-set-selections <<< "postfix postfix/mailname string your.hostname.com"
+sudo debconf-set-selections <<< "postfix postfix/main_mailer_type string 'Internet Site'"
+sudo debconf-set-selections <<< "archivematica-mcp-server archivematica-mcp-server/dbconfig-install boolean true"
+
+curl -s https://bootstrap.pypa.io/get-pip.py | sudo python -
+sudo apt-get update
+for item in "${packages[@]}"; do
+    sudo DEBIAN_FRONTEND=noninteractive apt-get install --yes ${item}
+done
+
+sudo rm -f /etc/nginx/sites-enabled/default
+sudo ln -sf /etc/nginx/sites-available/storage /etc/nginx/sites-enabled/storage
+sudo ln -sf /etc/nginx/sites-available/dashboard.conf /etc/nginx/sites-enabled/dashboard.conf
+sudo systemctl reload nginx
+
+for item in "${services[@]}"; do
+    start_service "${item}"
+done
```

Comment:
The `clamav-daemon` service needs the signature file to start. Sometimes it takes more than 1 minute since `clamav-freshclam` is started. I tested the following that check 5 times every 30 seconds whether this service is up:

```
counter_check_clamav=0
until $(sudo systemctl -q is-active clamav-daemon)  || [ ${counter_check_clamav} -ge 5 ]  ; do
        echo "Waiting 30s for clamav signatures"
        sleep 30
        sudo systemctl -q start clamav-daemon
        counter_check_clamav=$[${counter_check_clamav}+1]
done
```